src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -455,6 +457,25 @@ private static bool NeedsSeparator(SyntaxToken token, SyntaxToken next)
             return false;
         }
 
+        private static bool KeywordNeedsSeparatorBeforeOpenParen(SyntaxKind kind)
+        {
+            switch (kind)
+            {
+                case SyntaxKind.TypeOfKeyword:
+                case SyntaxKind.DefaultKeyword:
+                case SyntaxKind.NewKeyword:
```

Comment:
❓ Does this trigger on `new (int, int)()`?

Reply:
new followed by parens, that's not valid tuple syntax.  It's been reserved for target-typed new (new followed by parens with values).

> error CS8181: 'new' cannot be used with tuple type. Use a tuple literal expression instead.

Reply:
That said, that raises the question of whether we'll want `new(1, 2)` or `new (1, 2)`. I suspect the latter, but that's not yet been decided.
I added a [note](https://github.com/dotnet/csharplang/issues/100#issuecomment-350823105) to the proposal to remember to address this open question.

Reply:
>new followed by parens, that's not valid tuple syntax.

But we do have tuple arrays, `new(int, int)[]` not sure if `new (int, int)[]` is preferable though.

> new(1, 2) or new (1, 2)

I'd rather choose `new(..)` because we already have `new()` constraint which is formatted that way.

Reply:
Good point. Forgot about arrays